pylib/cqlshlib/cql3handling.py

```diff
@@ -1480,12 +1501,12 @@ def alter_type_field_completer(ctxt, cass):
              ;
 
 <createUserStatement> ::= "CREATE" "USER" ( "IF" "NOT" "EXISTS" )? <username>
-                              ( ("WITH" ("HASHED")? "PASSWORD" <stringLiteral>) | ("WITH" "GENERATED" "PASSWORD") )?
```

Comment:
please rebase, i dont get why this is removed

Reply:
fixed, creating a new PR